### ISSUE TITLE
Kohn-Sham equations as regularizer : building prior knowledge into machine-learned physics

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#130]
# TL;DR
In the problem of approximating physical simulations using density functional theory with neural networks, physical constraints can be imposed on ML models by treating the Kohn-Sham equation as a differentiable model. This greatly improves the accuracy of exchange-correlation term calculations.
![direct ML](https://user-images.githubusercontent.com/24524018/95158805-81160880-07d7-11eb-9e7b-db50a6388d94.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.08551

# Submission Dates(yyyy/mm/dd)
2020/09/17

# Authors and institutions
Li Li, Stephan Hoyer, Ryan Pederson, Ruoxi Sun, Ekin D. Cubuk, Patrick Riley, Kieron Burke
- Google Research, Mountain View, CA 94043, USA
- Department of Physics and Astronomy, University of California, Irvine, CA 92697, USA
- Department of Chemistry, University of California, Irvine, CA 92697, USA

# Methods

# Results

# Comments
